src/base/charstr.rs

```diff
@@ -56,6 +56,9 @@ use std::vec::Vec;
 pub struct CharStr<Octs: ?Sized>(Octs);
 
 impl<Octs: ?Sized> CharStr<Octs> {
+    /// Character strings have a maximum length of 255 octets.
+    pub const MAX_LENGTH: usize = 255;
```

Comment:
I’m now also wondering if the constant should be called `MAX_LEN`, given that the method is called `len`?

Reply:
okay fair enough 👍 done!